common/src/main/kotlin/io/homeassistant/companion/android/common/assist/AssistViewModelBase.kt

```diff
@@ -246,20 +264,45 @@ abstract class AssistViewModelBase(
         return true
     }
 
-    protected fun setupRecorderQueue() {
-        recorderQueue = mutableListOf()
+    /**
+     * Sets up audio recording and buffering for voice input.
+     *
+     * Must be called before [runAssistPipelineInternal] for voice pipelines.
+     * Audio data is buffered until [handleSttStart] signals that the server is ready
```

Comment:
You're referring to a private function in 'public' documentation

```suggestion
     * Audio data is buffered until the server is ready
```